static/js/deactivation-survey.js

```diff
@@ -1,190 +1,272 @@
-        const dialogHTML = `
-        <div class="nfd-deactivation-survey__overlay" nfd-deactivation-survey-destroy></div>
-        <div class="nfd-deactivation-survey__container" role="document">
-            <div class="nfd-deactivation-survey__content">
-                <h1 id="nfd-deactivation-survey-title" class="nfd-hidden" aria-hidden="true">${runtimeData.strings.surveyTitle}</h1>
-                <div class="nfd-deactivation-survey__content-header">
-                    <h3>${runtimeData.strings.dialogTitle}</h3>
-                    <p>${runtimeData.strings.dialogDesc}</p>
-                </div>
-                ${getSurveyFormHTML()}
-                <span class="nfd-deactivation-survey_loading nfd-hidden"></span>
+	// Data module runtime data
+	const runtimeData = window.newfoldDeactivationSurvey;
+	// Dialog instance / will be initialized later
+	let deactivationSurveyDialog;
+
+	const renderDialog = () => {
+		// Create dialog container
+		const surveyDialog = document.createElement( 'div' );
+		surveyDialog.id = 'nfd-deactivation-survey';
+		surveyDialog.setAttribute(
+			'aria-labelledby',
+			'nfd-deactivation-survey-title'
+		);
+		surveyDialog.setAttribute( 'aria-hidden', 'true' );
+		surveyDialog.innerHTML = getDialogHTML();
+
+		// Append dialog container to DOM
+		const wpAdmin = document.querySelector( 'body.wp-admin' );
+		wpAdmin.appendChild( surveyDialog );
+
+		// Disable body scroll
+		document.body.classList.add( 'nfd-noscroll' );
+
+		// Create dialog instance
+		deactivationSurveyDialog = new A11yDialog( surveyDialog );
+		deactivationSurveyDialog.show();
+	};
+
+	const getSureContent = () => {
+		const content = `
+			<div class="nfd-deactivation-sure nfd-deactivation__content">
+				<div class="nfd-deactivation__header">
+					<h1 class="nfd-deactivation__content-title">${ runtimeData.strings.sureTitle }</h1>
+                    <p class="nfd-deactivation__content-subtitle">This plugin is powers important features on your site. These will no longer be available if you deactivate the plugin.</p>
+				</div>
+				<div class="nfd-deactivation__body">
+                    <div class="nfd-deactivation__cards">
+						<div class="nfd-deactivation__card">Hosting integrations</div>
+						<div class="nfd-deactivation__card">Performance Improvements</div>
+						<div class="nfd-deactivation__card">Staging Site Support</div>
+						<div class="nfd-deactivation__card">Wonder Blocks</div>
+						<div class="nfd-deactivation__card">Integrated AI Help Center</div>
+					</div>
+				</div>
+				<div class="nfd-deactivation__footer"><div class="nfd-deactivation__footer-content">
+					<div class="nfd-deactivation-survey__content-actions">
+						<div class="nfd-deactivation__helptext" "style="text-align:left">
+							<p><b>Need Help?</b> Learn more about <a href="#" target="_blank" nfd-deactivation-survey-destroy>the features of this plugin</a>, check the <a href="#">help center</a>, or <a href="#">contact support</a>.</p>
```

Comment:
Same here with the copy, and the `href` doesn't seem to be going anywhere.

Reply:
Added placeholder urls, and will likely rework this section once we have copy.